recipes/r-splines2/meta.yaml

```diff
@@ -0,0 +1,85 @@
+{% set version = '0.3.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-splines2
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/splines2_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/splines2/splines2_{{ version }}.tar.gz
+  sha256: db53fa4050920f1890f6162b3fcb2bd3d89933700828a2e48e6000f0d11ef00f
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+  host:
+    - r-base
+    - r-rcpp
+    - r-rcpparmadillo
+  run:
+    - r-base
+    - {{ native }}gcc-libs         # [win]
+    - r-rcpp
+    - r-rcpparmadillo
+
+test:
+  commands:
+    - $R -e "library('splines2')"           # [not win]
+    - "\"%R%\" -e \"library('splines2')\""  # [win]
+
+about:
+  home: https://github.com/wenjie2wang/splines2
+  license: GPL-3.0-only
```

Comment:
# License: GPL (>= 3) is GPL-3.0-or-later:
```suggestion
  license: GPL-3.0-or-later
```

Reply:
Good call